dds_registration/settings.py

```diff
@@ -35,21 +35,32 @@
 environ.Env.read_env(os.path.join(BASE_DIR, ".env"))
 environ.Env.read_env(os.path.join(BASE_DIR, ".env.local"))
 
-DEV = env("DEV")
-DEBUG = env("DEBUG")
-LOCAL = env("LOCAL")
+
+def random_string(length: int = 32) -> str:
+    possibles = string.ascii_letters + string.digits
+    return "".join(random.sample(possibles, length))
+
+
+DEV = env("DEV") or env("DEBUG") or env("LOCAL")
+LOCAL = DEBUG = DEV
 
 SECRET_KEY = env("SECRET_KEY")
 REGISTRATION_SALT = env("REGISTRATION_SALT")
 SENDGRID_API_KEY = env("SENDGRID_API_KEY")
 
-if not SECRET_KEY or not REGISTRATION_SALT or not SENDGRID_API_KEY:
-    error_text = "Error: Environment configuration variables are required (check for your local `.env*` files, refer to `.env.SAMPLE`)."
-    raise Exception(error_text)
+SECRETS = [
+    (SECRET_KEY, "SECRET_KEY"),
+    (REGISTRATION_SALT, "REGISTRATION_SALT"),
+    (SENDGRID_API_KEY, "SENDGRID_API_KEY"),
+]
 
-if not SECRET_KEY or not REGISTRATION_SALT or not SENDGRID_API_KEY:
-    error_text = "Error: Environment configuration variables are required (check for your local `.env` file, refer to `.env.SAMPLE`)."
-    raise Exception(error_text)
+for key, label in SECRETS:
```

Comment:
These things look a bit unreasonably overcomplicated (for me). But that's ok if you think it's better or simpler.

Reply:
Giving an error message which helps you understand which environment variable is missing is much better than "oops, stuff died, sorry" 😛

Reply:
I'd (personally) prefer to test cases separately (if it's important) and throw exact exceptions for each of them than to have this (a bit) unobvious code.

Reply:
> I'd (personally) prefer to test cases separately (if it's important) and throw exact exceptions 

That's exactly what this does?